content/chainguard/chainguard-enforce/iam-groups/rolebinding-terraform-gh/index.md

```diff
@@ -0,0 +1,393 @@
+---
+title: "Create Rolebindings for a GitHub Team Using Terraform"
+linktitle: "GitHub Team Role Binding"
+description: "Procedural tutorial outlining how to use Terraform to create Chainguard rolebindings for members of a GitHub team."
+type: "article"
+date: 2023-06-10T08:48:45+00:00
+lastmod: 2023-06-10T08:48:45+00:00
+draft: false
+tags: ["Enforce", "Product", "Procedural"]
+images: []
+weight: 050
+---
+
+There may be cases where an organization will want multiple users to have access to the same Chainguard Enforce instance. Originally, Chainguard allowed you to grant other users access to Enforce by [generating an invite link or code](/chainguard/chainguard-enforce/iam-groups/how-to-manage-iam-groups-in-chainguard-enforce/#inviting-others-to-a-group).
+
+However, this process has its limitations, as it forces you to generate invite codes for each user one at a time. It also means that you must ensure that the invite codes or links are distributed in a secure manner, or else you risk unwanted users having access to Enforce.
+
+This guide outlines one method of using Terraform to grant members of a GitHub team access to the resources managed by a Chainguard Enforce group. It also highlights a few other Terraform configurations you can use to manage rolebindings in the Chainguard platform.
+
+
+## Prerequisites
+
+To complete this guide, you will need the following.
+
+* `terraform` installed on your local machine. Terraform is an open-source Infrastructure as Code tool which this guide will use to create various cloud resources. Follow [the official Terraform documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) for instructions on installing the tool.
+* `chainctl` — the Chainguard Enforce command line interface tool — installed on your local machine. Follow our guide on [How to Install `chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/) to set this up.
+* Access to a GitHub team. If you'd like, you can create a new GitHub organization and team for testing purposes. Check out [GitHub's documentation](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team) for details on how to do this. 
+
+> **Note**: The procedure outlined in this guide will only create rolebindings for members of the GitHub team that have already registered with Chainguard Enforce. To register with Chainguard under their GitHub account team members must [log into Chainguard Enforce](https://edu.chainguard.dev/chainguard/chainguard-enforce/authentication/log-in-chainguard-enforce/), making sure select **Sign In with GitHub** when signing in through their browser window and logging in through the appropriate GitHub account. Any members of the GitHub team that don't complete this will not have rolebindings created for them and they will not be able to access the associated resources.
```

Comment:
```suggestion
> **Note**: The procedure outlined in this guide will only create rolebindings for members of the GitHub team that have already registered with Chainguard Enforce. To register with Chainguard under their GitHub account team members must [log into Chainguard Enforce](https://edu.chainguard.dev/chainguard/chainguard-enforce/authentication/log-in-chainguard-enforce/), making sure they select **Sign In with GitHub** when signing in through their browser window and logging in through the appropriate GitHub account. Any members of the GitHub team that don't complete this will not have rolebindings created for them and they will not be able to access the associated resources.
```